setup.py

```diff
@@ -39,13 +39,13 @@ def find_version(*file_paths):
     install_requires.append('PyYAML>=3.10,<=3.13')
 
     # Colorama removed support for EOL pythons.
-    install_requires.append('colorama>=0.2.5,<0.3.9')
+    install_requires.append('colorama>=0.2.5,<=0.3.9')
 elif sys.version_info[:2] == (3, 3):
-    install_requires.append('PyYAML>=3.10,<=5.2')
+    install_requires.append('PyYAML>=3.10,<=3.13')
     # Colorama removed support for EOL pythons.
-    install_requires.append('colorama>=0.2.5,<0.3.9')
+    install_requires.append('colorama>=0.2.5,<=0.3.9')
 else:
-    install_requires.append('PyYAML>=3.10,<=5.2')
+    install_requires.append('PyYAML>=3.10,<5.2')
```

Comment:
guess there is no 5.2 yet, but the bound being `<` or `<=` should be consistent between setup.cfg and setup.py shouldn't it?

Reply:
Yeah good catch, I'll update that